paddle/fluid/pybind/pybind.cc

```diff
@@ -1325,6 +1325,8 @@ All parameter, weight, gradient are variables in Paddle.
       .def("_equals", &IsSamePlace<platform::CUDAPlace, platform::CPUPlace>)
       .def("_equals",
            &IsSamePlace<platform::CUDAPlace, platform::CUDAPinnedPlace>)
+      .def("_get_device_id",
```

Comment:
if only can used in gpu version, maybe add marco controling is better

Reply:
Done